openstack/identity/v2/tokens/requests.go

```diff
@@ -77,6 +78,10 @@ func (opts AuthOptions) ToTokenV2CreateMap() (map[string]interface{}, error) {
 	return b, nil
 }
 
+func (opts *AuthOptions) CanReauth() bool {
```

Comment:
nit: this method may not need to have a pointer receiver

Reply:
Agreed.